ejb-txn-remote-call/README.adoc

```diff
@@ -0,0 +1,1038 @@
+include::../shared-doc/attributes.adoc[]
+
+= ejb-txn-remote-call: Demonstrates remote EJB calls and transaction propagation
+:author: Ondra Chaloupka
+:level: Intermediate
+:technologies: EJB, JTA, Clustering
+
+[abstract]
+The `ejb-txn-remote-call` quickstart demonstrates remote EJB calls over two application servers of {productName}.
+
+:standalone-server-type: ha
+:archiveType: war
+:uses-h2:
+:requires-multiple-servers:
+:jbds-not-supported:
+
+== What is it?
+
+The `ejb-txn-remote-call` quickstart demonstrates the remote EJB calls over two application servers of {productNameFull}.
+
+== Description
+
+The EJB remote call contains a JTA transaction that is propagated over the EJB call.
+Further, the quickstart demonstrates the transaction recovery, which is run for both servers
+when a failure occurs.
+
+This quickstart contains two maven projects.
+The first maven project represents the caller side, and is intended to be deployed on the first server (`server1`).
+The second project represents the called side. This project is intended to be deployed
+to the other two servers (`server2` and `server2`). The two projects should not be deployed to the same server.
+
+.Maven projects in this quickstart
+[cols="40%,60%",options="headers"]
+|===
+|Project |Description
+
+|`client`
+|An application that you deploy to the first server, to `server1`.
+It provides REST endpoints that provides EJB remote calls to the `server application`
+residing on the other servers.
+Besides the remote EJB call the REST endpoint invocation processes data insertion to a database.
+As the REST invocation starts a transaction then the transaction enlists two participants
+(the database and the remote server called with EJB remote invocation).
+The transaction manager then uses the two-phase commit for committing the transaction.
+The quickstart examines failures and shows the transactional behaviour
+in such cases.
+
+|`server`
+|An application receives the remote EJB calls from the `client` application deployed at `server1`.
+This application is deployed to `server2` and `server3`.
+The EJB component receives the EJB remote call and depending on the scenario
+it resumes on the transaction propagated in the context of the remote EJB call.
+The call inserts data to the database.
+The applications also enlists a special XAResource to the transaction to demonstrate the quickstart.
+|===
+
+=== What to expect?
```

Comment:
Why not "Running the quickstart"?

Why is this heading nested a level lower than the previous heading?  Shouldn't they be at the same level?